ocs_ci/ocs/scale_lib.py

```diff
@@ -0,0 +1,385 @@
+        Args:
+            pods_per_iter (int): Number of PVC-POD to be created per PVC type
+                eg: If 2 then 8 PVC+POD will be created with 2 each of 4 PVC types
+            io_runtime (sec): Fio run time in seconds
+            start_io (bool): If True start IO else don't
+
+        Returns:
+            pod_objs (obj): Objs of all the PODs created
+            pvc_objs (obj): Objs of all the PVCs created
+        """
+        rbd_sc = helpers.default_storage_class(constants.CEPHBLOCKPOOL)
+        cephfs_sc = helpers.default_storage_class(constants.CEPHFILESYSTEM)
+        pvc_size = f"{random.randrange(5, 105, 5)}Gi"
+        logging.info(f"Create {pods_per_iter} PVCs and PODs")
+        cephfs_pvcs = helpers.create_multiple_pvc_parallel(
+            sc_obj=cephfs_sc, namespace=self.namespace, number_of_pvc=pods_per_iter,
+            size=pvc_size, access_modes=[constants.ACCESS_MODE_RWO, constants.ACCESS_MODE_RWX]
+        )
+        rbd_pvcs = helpers.create_multiple_pvc_parallel(
+            sc_obj=rbd_sc, namespace=self.namespace, number_of_pvc=pods_per_iter,
+            size=pvc_size, access_modes=[constants.ACCESS_MODE_RWO, constants.ACCESS_MODE_RWX]
+        )
+        # Appending all the pvc_obj and pod_obj to list
+        pvc_objs, pod_objs = ([] for i in range(2))
+        pvc_objs.extend(cephfs_pvcs + rbd_pvcs)
+
+        # Create pods with above pvc list
+        cephfs_pods = helpers.create_pods_parallel(
+            cephfs_pvcs, self.namespace, constants.CEPHFS_INTERFACE,
+            pod_dict_path=str(self.pod_dict_path), sa_name=self.sa_name.name,
+            dc_deployment=self.dc_deployment, node_selector=self.node_selector
+        )
+        rbd_rwo_pvc, rbd_rwx_pvc = ([] for i in range(2))
+        for pvc_obj in rbd_pvcs:
+            if pvc_obj.get_pvc_access_mode == constants.ACCESS_MODE_RWX:
+                rbd_rwx_pvc.append(pvc_obj)
+            else:
+                rbd_rwo_pvc.append(pvc_obj)
+        rbd_rwo_pods = helpers.create_pods_parallel(
+            rbd_rwo_pvc, self.namespace, constants.CEPHBLOCKPOOL,
+            pod_dict_path=str(self.pod_dict_path), sa_name=self.sa_name.name,
+            dc_deployment=self.dc_deployment, node_selector=self.node_selector
+        )
+        rbd_rwx_pods = helpers.create_pods_parallel(
+            rbd_rwx_pvc, self.namespace, constants.CEPHBLOCKPOOL,
+            pod_dict_path=str(self.pod_dict_path), sa_name=self.sa_name.name,
+            dc_deployment=self.dc_deployment, raw_block_pv=True,
+            node_selector=self.node_selector
+        )
+        temp_pod_objs = list()
+        temp_pod_objs.extend(cephfs_pods + rbd_rwo_pods)
+
+        # Appending all the pod_obj to list
+        pod_objs.extend(temp_pod_objs + rbd_rwx_pods)
+
+        fio_size = self.get_size_based_on_cls_usage()
+        fio_rate = self.get_rate_based_on_cls_iops()
+
+        # Start IO
```

Comment:
How do we find if there are any IO failures? Would the IO thread break?

Reply:
For now we don't have that capability, this is one of the biggest challenge in Scale Testing for now we don't have idea to identify IO failures. 
If any suggestions please do comment.

Reply:
Hmmm. I think `run_io` function should handle IO failures and we should make use of that functionality. 
@vasukulkarni Thoughts?

Reply:
Only do_verify is default, I think we should add verify=sha256 for atleast one job (or run it separately outside scale)

Reply:
But this PR doesn't seem to be the place, its has to be in different PR.